crosshair/main.py

```diff
@@ -94,6 +95,11 @@ def command_line_parser() -> argparse.ArgumentParser:
         metavar="FLOAT",
         help="Maximum seconds to spend checking execution paths for one condition",
     )
+    common.add_argument(
+        "--contract_file",
```

Comment:
One might want multiple files too?

It's interesting that a `contract_file` is very similar to a plugin - they are both just python files that we run at startup. What do you think about naming this `--extra_plugins`? In theory someone might want to conditionally apply `register_class()` via the command line, and conversely someone might use `register_contract()` via a plugin.

Reply:
Allowing multiple files is a good idea!

I have indeed been hesitating a lot on this name. I like the name `--extra_plugin`. The drawback is that it does not specify we are talking about contract registration. But at the same time, it's true that it might not only concern registering contracts; it could be used as well for the existing [plugins](https://crosshair.readthedocs.io/en/latest/plugins.html).

Reply:
> I have indeed been hesitating a lot on this name. I like the name `--extra_plugin`. The drawback is that it does not specify we are talking about contract registration. But at the same time, it's true that it might not only concern registering contracts; it could be used as well for the existing [plugins](https://crosshair.readthedocs.io/en/latest/plugins.html).

Yup! I believe it'll still make sense to have "how-to" documentation specific to contract registration, but at the level of the command line interface, I'm inclined to think of all these things as plugins.